packages/example-log-extension/src/types.ts

```diff
@@ -23,4 +23,6 @@ export type LevelMetadata = {level: number};
 
 export type HighResTime = [number, number]; // [seconds, nanoseconds]
 
+export type LogFn = (msg: string, level: number) => void;
```

Comment:
I am confused about the difference between `LogFn` and `LogAction`. Please add tsdoc comments to clarify.